src/main/java/cope/cosmos/client/features/modules/visual/LagESPModule.java

```diff
@@ -176,20 +197,20 @@ public void calculateIntermediary(){
             so normally i would use a higher order function for calcuting the interp,
             but im lazy and nobody will ever look at this. so i wont
              */
-            if( reverse.getValue()){
+            if (reverse.getValue()){
 
                 //calculate the difference
                 Vec3d d = from.subtract(to);
 
                 //set the position to be an interpolated value from `from` to `to`
-                if (fadeSetting.getValue() == fade.LINEAR) {
+                if (fadeSetting.getValue() == Fade.LINEAR) {
```

Comment:
.equals()

Reply:
.equals is literally just more overhead to check '=='
    public final boolean equals(Object var1) {
        return this == var1;
    }

Reply:
yeah i know but aesthetical told me to use it in one of my PRs (alt manager)